docs/examples/multiple-request-defineFetch.md

```diff
@@ -0,0 +1,102 @@
+# Multiple Request `defineFetch` Example
+
+One case Resift shines is when multiple requests need to be fired under one loading state.
+
+The use case below is an example for when series of people need to be deleted from a list.
+
+Example:
+
+```ts
+// component/actions/getPeople.ts
+
+import defineFetch from '@sift/resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Get People',
+  make: () => ({
+    key: [],
+    request: () => ({ http }) =>
+      return http({
+        method: 'GET',
+        route: `/people`,
+      }),
+  }),
+});
+```
+
+```ts
+// component/actions/deletePeople.ts
+import makeGetPeople from './getPeople';
+import defineFetch from '@sift/resift/defineFetch';
+
+export default defineFetch({
+  displayName: 'Delete People',
+  make: () => ({
+    key: [],
+    request: (peopleToDelete: person[]) => async ({ http, dispatch }) => {
+      const errors: Error[] = [];
+      await Promise.all(
+        peopleToDelete.map((id: string) => {
+          return http({
+            method: 'DELETE',
+            route: `/person/${id}`
+          }).catch((err: Error) => {
+            errors.push(err);
+          });
+        })
+      );
+
+      // After completed, fetchPeople again.
+      const getPeople = makeGetPeople();
+      dispatch(getPeople());
+
+      return errors.length ? errors : null;
```

Comment:
I don't think it's a good idea to return a list of errors here. In real usage, it might be better to throw if there are any errors.

Since this is an example, I think the easiest way to teach the concept is to just sequentially delete all the people using `async`, `await`, and a normal `for of` loop.

If there is any error, it will cause the whole fetch to stop and error status to show up

Reply:
Change it to how you would like. I have updated it to now log the error instead of returning all errors. 
![Screen Shot 2019-08-08 at 10 56 46 AM](https://user-images.githubusercontent.com/6955546/62713816-5958c400-b9cb-11e9-8532-c6803919ca69.png)